pkg/snapstore/abs_snapstore.go

```diff
@@ -116,6 +115,23 @@ func getCredentials(prefixString string) (string, string, error) {
 		return credentials.StorageAccount, credentials.SecretKey, nil
 	}
 
+	// TODO: @renormalize Remove this extra handling in v0.31.0
```

Comment:
I would suggest open a issue to drop the support for secret passing via JSON credential file, so that it become easy to track

Reply:
assign that issue a milestone of `v0.31.0`

Reply:
Yeah will be doing that. Thanks for the suggestion!

Reply:
I've created https://github.com/gardener/etcd-backup-restore/issues/741 to track this.